content/en/docs/faq/terminology.md

```diff
@@ -0,0 +1,65 @@
+of the commonly seen trust stores are similar. An example would be [Mozilla's CA Certificate Program](https://wiki.mozilla.org/CA).
+
+### What does "self-signed" mean? Is my CA self-signed?
+
+Self-signed means exactly what it says; a certificate is self-signed if it is signed by its own private key.
+
+Self-signed is a commonly confused term, however, and is very frequently misused to mean "not publicly trusted". We tend to use terms
+like "private PKI" to denote the situation where an organization might have their own internal CA certificates which wouldn't
+be trusted outside of the organization.
+
+As an example, there are _many_ self-signed certificates in [Mozilla's CA Certificate Program](https://wiki.mozilla.org/CA), but
+all of those certificates would usually be described as "publicly trusted".
+
+Your certificate is self-signed only if it's signed with its own key.
+
+### What's the difference between "root", "intermediate", and "leaf" certificates?
+
+cert-manager uses the following definitions:
+
+#### Root Certificates
+
+Roots are self-signed certificates and almost always marked as CA certificates. They're usually not sent over the wire
+during a TLS handshake because they need to be explicitly trusted in order to be validated.
+
+Roots are sometimes defined as "CA certificates which are explicitly trusted" - which can include certificates which
+aren't self-signed. cert-manager doesn't use this definition.
+
+Changing trust stores to include new roots or remove old ones is a non-trivial task which can take months or years for publicly
+trusted roots. For this reason, roots are usually issued with very long lifetimes - often on the order of decades.
+
+#### Intermediate Certificates
+
+Intermediates are CA certificates signed by another CA. Most intermediates will be signed by a root certificate, but it's
+possible to construct longer chains where an intermediate can be signed by another intermediate.
+
+Intermediate certificates are usually issued with a much shorter lifetime than the CA which signed them. On the public
+internet, intermediate certificates are used on network-connected machines for day-to-day issuance so that the
+highly-valuable root certificates can remain entirely offline.
+
+While intermediate certificates can also be explicitly trusted via addition to a trust store, they're usually validated
+by "walking up" the chain and validating signatures until an explicitly trusted self-signed root certificate is found.
+
+#### Leaf Certificates
+
+Leaf certificates are usually not CA certificates, and are used to represent a particular identity; on the public internet,
+leaf certificates usually identify a particular domain, such as `example.com`.
+
+Leaf certificates are generally at the end of a chain of certificates, and must be sent along with one or more intermediates
+to create a chain which can be validated by verifying signatures up to a trusted root certificate.
```

Comment:
```suggestion
Leaf certificates are never CA certificates, and are used to represent a particular identity; on the public internet,
leaf certificates usually identify a particular domain, such as `example.com`.

Leaf certificates are always at the end of a chain of certificates, and must be sent along with one or more intermediates
to create a chain which can be validated by verifying signatures up to a trusted root certificate.
```
I referred to: https://en.wikipedia.org/wiki/Public_key_certificate#End-entity_or_leaf_certificate and I know Wikipedia is pretty unreliable but is there anywhere that describes a CA leaf certificate.

I mean does leaf / root have a different meaning when talking about a certificate bundle (e.g. tls.crt) as opposed to certificates used by clients and servers on the Internet. Is a leaf certificate, one without any children in a particular bundle. 
Is a root certificate  any self-signed certificate OR a certificate who's parent is not included in the same bundle?

Reply:
> I referred to: https://en.wikipedia.org/wiki/Public_key_certificate#End-entity_or_leaf_certificate and I know Wikipedia is pretty unreliable but is there anywhere that describes a CA leaf certificate.

> Is a leaf certificate, one without any children in a particular bundle.

I can't think of why anyone would have a CA leaf certificate, I just know it's _technically_ possible. Probably too far out there to be worth mentioning; I'll remove that bit.

I think you could make the case that such a thing could exist, but I don't know why anyone would do that really and I probably shouldn't have accounted for it in this page.